Ds3/Runtime/CustomPercentEscaper.cs

```diff
@@ -0,0 +1,75 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ds3.Runtime
+{
+    internal static class CustomPercentEscaper
+    {
+        private static readonly char[] _hexChars = new char[] {
```

Comment:
static var should start with Capital => 
private static readonly char[] HexChars = {
            '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'
        };